sklearn/svm/src/libsvm/svm.cpp

```diff
@@ -1816,6 +1816,118 @@ static void solve_nu_svr(
 	delete[] y;
 }
 
+static void solve_svdd_l1(
```

Comment:
Can you please insert the information from https://github.com/scikit-learn/scikit-learn/pull/7910#issuecomment-580457001 as an inline comment in the source code? At the top of this file there is a comment by Fabian from 2010 that could be extended with specific information about SVDD-L1.